staging/src/k8s.io/client-go/rest/request.go

```diff
@@ -714,6 +715,11 @@ func (r *Request) Do() Result {
 
 	var result Result
 	err := r.request(func(req *http.Request, resp *http.Response) {
+		rest_log := fmt.Sprintf("Request: %s %s\n", req.Method, req.URL)
```

Comment:
Also I think the assumption here is we always use client go in our system. While that is true right now, it may not be true later and it is not definitely a requirement for conformance. For example we may write a controller with python client and package it as part of the system.

I suggest you do it server side. We also should not do this logging all the time as it could get big and costly. We need to think on how to enable this for kubernetes instances that we want to assess for conformance. Just thinking out loud: runtime flag?!